### PR TITLE
Azure & Cirrus CI: Bump macOS image to 11.x (Big Sur) [2]

### PR DESCRIPTION
**Pull request by kinke**

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -200,12 +200,13 @@ task:
...
 task:
-  name: macOS 10.15 $TASK_NAME_SUFFIX
+  name: macOS 11.x $TASK_NAME_SUFFIX
   osx_instance:
-    image: catalina-xcode
+    image: big-sur-xcode
   timeout_in: 60m
   environment:
     CI_OS: osx
+    EXTRA_CMAKE_FLAGS: "-DD_COMPILER_FLAGS=-gcc=/usr/bin/cc"
```
Comment (JohanEngelen) [inline .cirrus.yml:209]: :))

Comment (JohanEngelen) [reply .cirrus.yml:209]: If this works, this is a good tip to add to the readme on how to build LDC on macOS.

**Comment by kinke** [comment]
Superseded by #3951 and #3952.

File: .cirrus.yml (hunk 1 above)
Comment (kinke) [reply .cirrus.yml:209]: I've added a note in https://wiki.dlang.org/Building_LDC_from_source#Building_LDC_from_source.

**Pull request closed without merging**